query-engine/connectors/mongodb-query-connector/src/interface/transaction.rs

```diff
@@ -201,17 +201,28 @@ impl<'conn> WriteOperations for MongoDbTransaction<'conn> {
         .await
     }
 
-    async fn execute_raw(&mut self, _inputs: HashMap<String, PrismaValue>) -> connector_interface::Result<usize> {
-        Err(MongoError::Unsupported("Raw queries".to_owned()).into_connector_error())
+    async fn execute_raw(&mut self, inputs: HashMap<String, PrismaValue>) -> connector_interface::Result<usize> {
+        catch(async move { write::execute_raw(&self.connection.database, &mut self.connection.session, inputs).await })
+            .await
```

Comment:
This code is currently unreachable, all three of `findRaw`, `rawCommandRaw` and `aggregateRaw` are parsed as `WriteQuery::QueryRaw`.

If that is suddenly changed but `write::execute_raw` is not implemented, then
* before this change QE would
   * panic outside of iTX
   * return an error inside of iTX
* and after this change it would
  * panic outside of iTX
  * panic inside of iTX

Which behavior is more preferable? I can revert this change and make `MongoDbConnection` return an error here instead.

Reply:
Changed `root_queries::write::execute_raw` to return `Err(MongoError::Unsupported("execute_raw".into()))` instead of `unimplemented!()` as per discussion in slack